src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/Join.txt

```diff
@@ -0,0 +1,189 @@
+﻿#SETUP: PowerFxV1CompatibilityRules,StronglyTypedBuiltinEnums, SupportColumnNamesAsIdentifiers
+
+// Inner
+>> Join(
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

we should have some tests passing blank, error arguments. To verify those are being handled.

Reply:
also - a case creating conflicts via 'As' 

..., F1 As R1, F2 As R1) // force conflict 

..., F1 As F2, F2) // implicit conflict.